dvc/system.py

```diff
@@ -21,55 +22,19 @@ def copy(src, dest):
 
     @staticmethod
     def hardlink(source, link_name):
-        import ctypes
-        from dvc.exceptions import DvcException
-
         source, link_name = fspath(source), fspath(link_name)
-
-        if System.is_unix():
-            try:
-                os.link(source, link_name)
-                return
-            except Exception as exc:
-                raise DvcException("link") from exc
-
-        CreateHardLink = ctypes.windll.kernel32.CreateHardLinkW
-        CreateHardLink.argtypes = [
-            ctypes.c_wchar_p,
-            ctypes.c_wchar_p,
-            ctypes.c_void_p,
-        ]
-        CreateHardLink.restype = ctypes.wintypes.BOOL
-
-        res = CreateHardLink(link_name, source, None)
-        if res == 0:
-            raise DvcException("CreateHardLinkW") from ctypes.WinError()
+        try:
+            os.link(source, link_name)
+        except OSError as exc:
+            raise DvcException("failed to link") from exc
 
     @staticmethod
     def symlink(source, link_name):
```

Comment:
@efiop , python 3 added several arguments to `hardlink` and  `symilnk`, should we support them?

`os.symlink(src, dst, target_is_directory=False, *, dir_fd=None)`

should it be appropriate to use `*args, **kwargs` in that case? (since it is only a wrapper)

Reply:
@mroutis We don't use them so no need to bother.